### PR TITLE
Requirement

### PR DESCRIPTION
**Pull request by I-Hong**
This PR is to fix the version of the requirement, https://github.com/IBM/MAX-Facial-Age-Estimator/issues/27#issue-435916513 and https://github.com/IBM/MAX-Facial-Age-Estimator/issues/26#issue-435907026

### CODE REVIEW

File: api/predict.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
 label_prediction = MAX_API.model('LabelPrediction', {
     'age_estimation': fields.Integer(required=True, description='Estimated age for the face'),
-    'face_box': fields.List(fields.Integer(required=True), description='Bounding box coordinates for the face')
+    'detection_box': fields.List(fields.Float(required=True), description='Bounding box coordinates for the face, ' + \
+         'in the form of an array of normalized coordinates [xmin, ymin, xmax, ymax]. Each coordinate is in the range [0, 1]')
```
Comment (bdwyer2) [inline api/predict.py]: In the README you said the order is `[ymin, xmin, height, width]`. Which is the correct order?

File: tests/test.py [hunk 2]
```diff
@@ -67,24 +67,5 @@ def test_predict():
         r = requests.post(url=model_endpoint, files=file_form)
     assert r.status_code == 400
 
-def test_img_resize():
```
Comment (bdwyer2) [inline tests/test.py]: Do you plan on re-adding this test back in the future? Maybe you could comment it out and/or add a TODO comment instead?

File: api/predict.py (hunk 1 above)
Comment (I-Hong) [reply api/predict.py]: Thanks @bdwyer2. The format is now consistent with our models with [ymin, xmin, height, width].

File: tests/test.py (hunk 2 above)
Comment (I-Hong) [reply tests/test.py]: Do we need to add a test for this (I'm planing to remove the different sizes of images)? Any suggestions?

Comment (bdwyer2) [reply tests/test.py]: If you don't think it's needed than this is fine

File: api/predict.py (hunk 1 above)
Comment (bdwyer2) [reply api/predict.py]: Okay, then you should change [README.md#5](https://github.com/IBM/MAX-Facial-Age-Estimator/pull/28/files#diff-04c6e90faac2675aa89e2176d2eec7d8R5)

Comment (bdwyer2) [reply api/predict.py]: So you will change this?
```suggestion
         'in the form of an array of normalized coordinates [ymin, xmin, height, width]. Each coordinate is in the range [0, 1]')
```

Comment (I-Hong) [reply api/predict.py]: Yes

**Pull request closed without merging**